### PR TITLE
[LDC] update travis.yml

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

**Comment by 9il** [comment]
Failed because #19.

**Comment by kyllingstad** [comment]
I'm not very familiar with Travis yet. Is it possible to specify just DMD and LDC as compilers, rather than specific versions? With this change, I have to remember to update `travis.yml` every time a new compiler version is released, if I want to test the code with the latest version.

**Comment by 9il** [comment]
ping @MartinNowak
I have tried to write `  - dmd` but it seems there is an bug in Travis-CI:
```
The command "curl  > ~/dmd.zip" failed and exited with 2 during .
```
If you forget to update `.travis.yml ` then travis will use old version of compiler.
In 99% cases if Travis success with old version then it will success with newer one.

**Comment by MartinNowak** [comment]
For now only compilers with specific versions are supported (e.g. dmd-2.066.1). This will remain the case until there is a generic way to obtain the latest version of each compiler. The default compiler is dmd-2.066.1. It's often a good idea to maunally update the compiler version, so that you can handle deprecations and such.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 language: d
+os:
+  - linux
+  - osx
```
Comment (MartinNowak) [inline .travis.yml]: Why do you need to test OSX. That wouldn't work with apt-get anyhow.

Comment (9il) [inline .travis.yml]: When (if) Travis will support OS X `apt-get` raise an error like reminder.

Comment (kyllingstad) [inline .travis.yml]: Yes, but that error will be completely unrelated to the pull request that triggers it, and that is unfortunate. If Travis does not support D on OS X, the `os:` section should just be removed.

Comment (9il) [inline .travis.yml]: Removed

**Comment by kyllingstad** [comment]
Ok, I'll merge this as soon as the LDC issue is sorted out and the build succeeds. Thanks!

File: .travis.yml (hunk 1 above)
Comment (MartinNowak) [inline .travis.yml]: It already does support D on OSX, but there isn't anything platform specific in this library, so you shouldn't needlessly waste Travis' resources.

**Comment by 9il** [comment]
This PR is a part of #22

**Pull request closed without merging**